### ISSUE TITLE
[2021 arXiv] ResNet strikes back: An improved training procedure in timm

### ISSUE THREAD

**Issue by gusdlf93** [gusdlf93/Paper_Survey#1]
Resnet은 2015년에 나온 후로, 많은 논문들에서 새로운 방법을 제시할 때, baseline으로서 사용이 되어 왔습니다.
이는 Resnet이 지닌 심플함과 안정적인 성능 때문일 겁니다.
시간이 지남에 따라서 많은 모델들이 나왔고, 많은 방법론들이 생겼습니다만,
이러한 논문들이 Resnet과 비교해서 얼마나 성능이 뛰어난지에 대해서는 공정하게 비교되지 않은것 같습니다.
Batch size가 다르거나, optimizer가 다르거나, Augmentation이 다르거나...
우리는 Resnet에서 학습과정에서 제어할 수 있는 하이퍼 파라미터의 변화만을 통해서 어느정도까지 성능이 올라 갈 수 있는 지에 대해서 알아보고자 합니다.

Epoch : 100, 300, 600
Resolution : 160, 224
Batchsize : 2048
Seed : 1~100까지 100번 실험
Data Augmentation : Random Resized Crop, horizontal flip, Mixup, CutMix
Regularization : weight decay, label smoothing, **Repeated-Augmentation, stochastic-Depth**
Optimization : SGD, **AdamW, LAMB**

![캡처](https://user-images.githubusercontent.com/89114114/136666859-4fa2ea5d-a6e1-4d66-9978-875544ff011f.PNG)

해당 학습 방법은 다른 네트워크들에서도 상당한 성능의 향상을 보여준다고 주장하고 있는데, 잘 설계된 좋은 네트워크일수록, 성능의 향상폭이 적으며, 심지어 ViT나 Efficient Net 기준으로는 Original이 훨씬 더 좋은 성능을 보여주고 있습니다.
저자는, 더 깊은 모델이거나 Architecture의 구조가 상당히 다르면, 그에 따라서 하이퍼 파라미터 또한 수정이 필요하다고 주장하고 있습니다. 이는 모델에 따라서 하이퍼 파라미터 탐색이 필요하다고 주장하고 있는 것입니다.
![그림2](https://user-images.githubusercontent.com/89114114/136667579-7801b785-bad7-41ee-9d07-8113bcc51c43.png)

대부분의 모델들은 학습 초기에 성능의 변이가 큽니다. - 이부분은 굳이 실험을?
![그림3](https://user-images.githubusercontent.com/89114114/136667791-9a771b25-37c4-4a2a-9ca6-ab6c58aaed8d.PNG)
